tests/test_reader.py

```diff
@@ -132,13 +130,25 @@ def test_reader_handles_duplicate_columns(tmp_hyper):
     assert df.columns.tolist() == ["col", "col_1", "col_2"]
 
 
-def test_frame_from_hyper_doesnt_generate_hyperd_log(frame, tmp_hyper):
+def test_frame_from_hyper_doesnt_generate_hyperd_log(
+    frame,
+    tmp_hyper,
+    tmp_path,
+    monkeypatch,
+):
+    monkeypatch.chdir(tmp_path)
     pt.frame_to_hyper(frame, tmp_hyper, table="test")
     pt.frame_from_hyper(tmp_hyper, table="test")
-    assert not pathlib.Path("hyperd.log").is_file()
+    assert not (tmp_path / "hyperd.log").is_file()
```

Comment:
I don't think there is anything that guarantees tmp_hyper and tmp_path are the same directory, so I'm not sure this really tests anything.

When running the test suite locally I would still see some tests generating a hyperd.log (not the ones you fixed) so I think there is something else in our test suite / code base that generates this outside of the work you already did

Reply:
I used tmp_path here to change the working directory to a temporary one, so that if a hyperd.log is generated, it does not persist in the file system across other tests; conversely, if other tests generate a hyperd.log file, the assert does not fail because of the file being present _before_ the test runs (at least, that's the idea, hopefully it works as intended).

Indeed, every test that uses a HyperProcess without the parameter `{"log_config": ""}` generates a hyperd.log file. That's the case of test_reads_nullable_columns in test_reader.py, and of many others.

Reply:
To my knowledge, there's no way to disable it globally and the parameter must be passed every time a HyperProcess is instantiated (but I might (and hope to) be wrong). But that's right, disabling the configuration in every test should do it.